src/views/SettingsView.vue

```diff
@@ -192,22 +206,11 @@ const createOrUpdateUser = () => {
 						<div class="label py-0">
 							<span class="label-text">{{ startCase(lowerCase(r)) }}</span>
 						</div>
-						<div class="flex items-center gap-2">
-							<div v-for="p in l" :key="p" class="form-control">
-								<label class="label cursor-pointer gap-2 py-1">
-									<input :checked="currentUser?.permissions?.[r]?.includes(p)" type="checkbox"
-										class="checkbox-primary checkbox checkbox-xs rounded" @click="() => {
-												if (currentUser?.permissions?.[r]?.includes(p)) {
-													currentUser.permissions[r] = currentUser.permissions[r].filter((perm: string) => perm !== p)
-												} else if (currentUser?.permissions?.[r]) {
-													currentUser!.permissions![r].push(p)
-												} else {
-													currentUser!.permissions![r] = [p]
-												}
-											}" />
-									<span class="label-text">{{ p }}</span>
-								</label>
-							</div>
+						<div class="label py-0">
+							<v-select :options="filteredOptions(r)" v-model="(currentUser?.permissions ?? {})[r]"
```

Comment:
This `v-model` doesn't convince me, as it may be undefined it would not maintain the binding. Try use a writable computed ref to do it instead.